### PR TITLE
Finishlinelab3

### PR DESCRIPTION
**Pull request by AndrewBanin**
SUMMARY:
Cft modified

### CODE REVIEW

File: Finishlinelab3/finishlinelab3infra.yaml [hunk 1]
```diff
@@ -24,57 +28,50 @@ Parameters:
...
-        - IpProtocol: tcp
-          FromPort: '22'
-          ToPort: '22'
-          CidrIp: !Ref SSHLocation
+      ImageId: !Ref AmiId 
+      Tags:
+        - Key: Name
+          Value: BaninaWebServer
+  BaninaWebServerSecurityGroup:
+  Type: AWS::EC2::SecurityGroup
```
Comment (ndzohdong) [inline Finishlinelab3/finishlinelab3infra.yaml]: Poor indentation. Place indent correctly

File: Finishlinelab3/finishlinelab3infra.yaml [hunk 2]
```diff
@@ -24,57 +28,50 @@ Parameters:
...
+      FromPort: '80'
+      ToPort: '80'
+      CidrIp: 0.0.0.0/0
+    - IpProtocol: tcp
+      FromPort: '22'
+      ToPort: '22'
+      CidrIp: 0.0.0.0/0          
+  BaninaDBSecurityGroup:
+  Type: AWS::RDS::DBSecurityGroup
+  Properties:
```
Comment (ndzohdong) [inline Finishlinelab3/finishlinelab3infra.yaml]: Poor indentation please fix it

File: Finishlinelab3/finishlinelab3infra.yaml [hunk 3]
```diff
@@ -24,57 +28,50 @@ Parameters:
...
+    DBSecurityGroupIngress:
+      EC2SecurityGroupName:
+        Ref: WebServerSecurityGroup
```
Comment (ndzohdong) [inline Finishlinelab3/finishlinelab3infra.yaml]: sysntax error should be
`!Ref WebServerSecurityGroup`

File: Finishlinelab3/finishlinelab3infra.yaml [hunk 4]
```diff
@@ -24,57 +28,50 @@ Parameters:
...
+      CidrIp: 0.0.0.0/0
+    - IpProtocol: tcp
+      FromPort: '22'
+      ToPort: '22'
+      CidrIp: 0.0.0.0/0          
+  BaninaDBSecurityGroup:
+  Type: AWS::RDS::DBSecurityGroup
+  Properties:
+    DBSecurityGroupIngress:
+      EC2SecurityGroupName:
```
Comment (ndzohdong) [inline Finishlinelab3/finishlinelab3infra.yaml]: Where is your database port? Please work with Corker to fix this

File: Finishlinelab3/finishlinelab3infra.yaml [hunk 5]
```diff
@@ -24,57 +28,50 @@ Parameters:
...
     Description: Public DNSName of the newly created EC2 instance
     Value: !GetAtt 
-      - EC2Instance
+      - BaninaWebserver
       - PublicDnsName
-  PublicIP:
-    Description: Public IP address of the newly created EC2 instance
-    Value: !GetAtt 
-      - EC2Instance
-      - PublicIp
```
Comment (ndzohdong) [inline Finishlinelab3/finishlinelab3infra.yaml:80]: Please us this in place of PublicDNS. Work with Corker and Charity

**Review by ndzohdong — CHANGES_REQUESTED** [review]

File: Finishlinelab3/finishlinelab3infra.yaml (hunk 1 above)
Comment (AndrewBanin) [reply Finishlinelab3/finishlinelab3infra.yaml]: done

File: Finishlinelab3/finishlinelab3infra.yaml (hunk 2 above)
Comment (AndrewBanin) [inline Finishlinelab3/finishlinelab3infra.yaml]: done

File: Finishlinelab3/finishlinelab3infra.yaml (hunk 3 above)
Comment (AndrewBanin) [reply Finishlinelab3/finishlinelab3infra.yaml]: Done

File: Finishlinelab3/finishlinelab3infra.yaml (hunk 5 above)
Comment (AndrewBanin) [reply Finishlinelab3/finishlinelab3infra.yaml:80]: Done

**Pull request merged**